### ISSUE TITLE
mailmerge broken starting from documentserver v7.2

### ISSUE THREAD

**Issue by aol-nnov** [ONLYOFFICE/DocumentServer#2134]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DocuemntServer/issues?q=) and did not find an issue describing my bug.

### Operating System of DocumentServer

Docker

### Version information

7.2, 7.3

### Expected Behavior

The data and document template are combined together resulting in a merged document

### Actual Behavior

document template is returned as mailmerge result. fields are not substituted

### Reproduction Steps

please find the minimal example attached

### Additional information

v7.1 works as expected

**Comment by aol-nnov** [comment]
#2128 may be related to this one

**Comment by XDmitryK** [comment]
This fixed in version 7.3.3 or later, issue 61300 and 59709. If this problem still exists, you can reopened this issue.

**Comment by Rita-Bubnova** [comment]
I close this issue. Feel free to comment or reopen it if you got further questions.

**Comment by aol-nnov** [comment]
Hello!

I've just checked against the following official docker images (onlyoffice/documentserver from hub.docker com)
* tag `7.3.3` Version: 7.3.3. Build: 49
* tag `7.4.1` Version: 7.4.1. Build: 36
* tag `7.5.1`, `latest` Version: 7.5.1. Build: 23

And I can confirm that this issue has been resolved. Thank you so much!

Nevertheless, #2135 is a real show-stopper now!.. I'd be extremely glad if that one will be resolved any time soon too!

Worth to mention that while checking this issue I've also checked #2135 with actual desktop editors version: 7.5.1 (543) x86_64 on MacOS 13.4.1
